regression/contracts-dfcc/loop_contracts_statement_expression/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+--dfcc main --apply-loop-contracts --disable-side-effect-check
```

Comment:
naming the option like could be confusing to the user, the side effect checking is not disabled, it is still happening at some point.

Reply:
The purpose of this flag is to disable side-effect checks now for the sake of experimenting other features, e.g., kani loop contracts and statement-expression loop contracts while we develop the new side-effect checks.